.vimrc

```diff
@@ -68,14 +115,14 @@
 " }
 
 " Use before config if available {
-    if filereadable(expand("~/.vimrc.before"))
-        source ~/.vimrc.before
+    if filereadable(g:spf13_vimrc_before)
+        exec 'source ' . g:spf13_vimrc_before
```

Comment:
if it's ok to use registers, then you can use source expand(@someregister) or so I'm told